lighttpd.conf

```diff
@@ -0,0 +1,46 @@
+mimetype.assign = (
+  ".html" => "text/html",
+  ".htm" => "text/html",
+  ".shtml" => "text/html",
+  ".css" => "text/css",
+  ".gif" => "image/gif",
+  ".jpeg" => "image/jpeg",
+  ".jpg" => "image/jpeg",
+  ".js" => "application/javascript",
+  ".txt" => "text/plain",
+  ".png" => "image/png",
+  ".ico" => "image/x-icon",
+  ".svg" => "image/svg+xml",
+  ".webp" => "image/webp",
+  ".json" => "application/json",
+)
+
+server.username = "www-data"
```

Comment:
don't we actually need to set a (matching) username in Dockerfile, too? 🤔

Reply:
just read about "Omit server.username to run lighttpd under the user account which started the lighttpd process."

will drop overwrite in config file so we can control user via a central place (dockerfile)